plutus-contract/src/Language/Plutus/Contract/Wallet.hs

```diff
@@ -166,28 +162,6 @@ addOutputs pk vl tx = tx & over Tx.outputs (pko :) where
 
 -- | Balance an unabalanced transaction, sign it, and submit
 --   it to the chain in the context of a wallet.
-handleTx :: (MonadWallet m, ChainIndexAPI m) => SigningProcess -> UnbalancedTx -> m Tx
-handleTx p utx =
-    balanceWallet utx >>= addSignatures p (Set.toList $ unBalancedTxRequiredSignatories utx) >>= WAPI.signTxAndSubmit
-
--- | The signing process gets a finished transaction and a list of public keys,
---   and signs the transaction with the corresponding private keys.
-newtype SigningProcess = SigningProcess { addSignatures :: forall m. (WalletAPI m, MonadError WalletAPIError m) => [L.PubKeyHash] -> Tx -> m Tx }
-
--- | The default signing process is 'signWallet'
-defaultSigningProcess :: SigningProcess
-defaultSigningProcess = signWallet
-
--- | Sign the transaction by calling 'WAPI.signTxnWithKey' (throwing a
---   'PrivateKeyNotFound' error if called with a key other than the
---   wallet's private key)
-signWallet :: SigningProcess
-signWallet = SigningProcess $
-    \pks tx -> foldM WAPI.signTxnWithKey tx pks
-
--- | Sign the transaction with the private keys of the given wallets,
---   ignoring the list of public keys that the 'SigningProcess' is passed.
-signWallets :: [Wallet] -> SigningProcess
-signWallets wallets = SigningProcess $ \_ tx ->
-    let signingKeys = E.walletPrivKey <$> wallets in
-    pure (foldr Tx.addSignature tx signingKeys)
+handleTx :: (MonadWallet m, ChainIndexAPI m, SigningProcessAPI m) => UnbalancedTx -> m Tx
+handleTx utx =
+    balanceWallet utx >>= addSignatures (Set.toList $ unBalancedTxRequiredSignatories utx) >>= WAPI.signTxAndSubmit
```

Comment:
Is this a typo?

Reply:
What do you mean is the typo?

Reply:
I meant the `unBalancedTxRequiredSignatories` function.